NextDom/nextdom-core
# [EN COURS] fix issue 1666 : modales

fix #1666
Add blur when modale

To test :
- check if height of modals is ok now with height résolution change 
- check all the update page (modale log, update counters etc)
- check all the backup page (modal log, backup, restore)
- test modale icon selector
- check the scenarios' log modals

NE PAS MERGER
